pkg/lib/service.js

```diff
@@ -253,7 +254,7 @@ export function proxy(name, kind) {
     systemd_manager.addEventListener("JobRemoved", on_job_new_removed_refresh);
 
     function wait(callback) {
-        wait_callbacks.promise.then(callback);
+        wait_promise.then(callback);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)